### PR TITLE
a server should fall back to a full handshake

### PR DESCRIPTION
**Pull request by kazu-yamamoto**
For #836, I would open this PR (again).

Please discuss whether or not we should use "SHOULD" here.

### CODE REVIEW

File: draft-ietf-tls-tls13.md [hunk 1]
```diff
@@ -2612,6 +2612,12 @@ facilitates implementation as described below). Servers MUST check
 that it is the last extension and otherwise fail the handshake with an
 "illegal_parameter" alert.
 
+If a server fails to retrieve a PSK value from identity values
+(because of some configuration or state changes, for instance), the
+server cannot resume the previous session nor verify the binders. In
+this case, the server SHOULD fall back to a full 1-RTT handshake if a
+"key_share" extension is supplied.
```
Comment (martinthomson) [inline draft-ietf-tls-tls13.md]: It might be a good idea to note that if the keys CAN be recovered, failing to verify the binders is fatal.

Comment (kazu-yamamoto) [inline draft-ietf-tls-tls13.md]: OK. I will do. Thanks.

Comment (kazu-yamamoto) [inline draft-ietf-tls-tls13.md]: Done.

**Comment by kazu-yamamoto** [comment]
The binder is validated with a key derived from the PSK value.
So, some readers might feel that "before using that PSK" is contradictory.

**Comment by ekr** [comment]
I just pushed a commit which I think resolves the ambiguity satisfactorily.

**Pull request closed without merging**